code/modules/vending/wardrobes.dm

```diff
@@ -269,7 +269,7 @@
 		/obj/item/clothing/glasses/regular = 2,
 		/obj/item/clothing/glasses/regular/jamjar = 1,
 		/obj/item/storage/bag/books = 1,
-		/obj/item/radio/headset/headset_srv = 2,
+		/obj/item/radio/headset/headset_srvmed = 2,
```

Comment:
?

Reply:
These are in the vending machines for replacements/job swaps. You can buy the normal med headset, pop out the encryption key out, and insert it into the service headset key and get the same results.

Coroners get a discount for both vendors, so why make the process of gearing up more difficult than necessary? Or make it harder for coroners to acquire roundstart equipment replacements?

Reply:
Oh this is the wrong vendor. My b.